Developer-Guide.md

```diff
@@ -63,15 +68,29 @@ $ sudo kata-runtime kata-check
 
 If your system is *not* able to run Kata Containers, the previous command will error and explain why.
 
-## Disable initrd
+## Configure to use initrd or rootfs image
 
-Unless you use an initial ramdisk (`initrd` or `initramfs`):
+Kata containers can run with either an initrd image or a rootfs image.
+
+If you want to test with `initrd`, make sure you have `initrd = /usr/share/kata-containers/kata-containers-initrd.img`
+in `/usr/share/defaults/kata-containers/configuration.toml` and remove the `image` line with e.g.,
```

Comment:
in `/usr/share/defaults/kata-containers/configuration.toml` and remove the `image` line with the following: